doc/tutorials/make_your_first_quantum_circuit.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<ul><li>discrete variable and continuous variable should not be capitalized</li><li>For links, make sure to do relative links instead of absolute links. For example, [default.qubit](../plugins.html#plugins)</li></ul><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
My relative links are not working...